artifacts/definitions/Windows/Forensics/RecentApps.yaml

```diff
@@ -37,12 +37,11 @@ sources:
             SELECT OSPath FROM glob(globs=UserHomes)
          },
          query={
-            SELECT AppId, AppPath, LaunchCount,
-                   timestamp(winfiletime=LastAccessedTime) AS LastExecution,
-                   timestamp(winfiletime=LastAccessedTime).Unix AS LastExecutionTS,
-                   parse_string_with_regex(
-                      string=Key.OSPath,
-                      regex="/Users/(?P<User>[^/]+)/ntuser.dat").User AS User
+            SELECT timestamp(winfiletime=LastAccessedTime) AS LastExecution,
+                   timestamp(winfiletime=LastAccessedTime).Unix AS _LastExecutionTS,
+                   OSPath as _OSPath,
+                   parse_string_with_regex(string=lowcase(string=OSPath),regex="c:\\\\users\\\\(?P<User>.+)\\\\ntuser.dat").User AS User,
```

Comment:
The correct way to extract the username is just `OSPath[2]`

Reply:
It's not updated on current versions. But we've got old server 2016s that havent been updated. 
I should update my blog post, I only tested win10 but turns out it's in some server versions as well

Fine to move it under reghunter, haven't played with that so don't know how it works.